src/host/readDataCooked.cpp

```diff
@@ -1243,12 +1291,6 @@ COOKED_READ_DATA::LayoutResult COOKED_READ_DATA::_layoutLine(std::wstring& outpu
     }
 
 outerLoopExit:
-    if (it != end && column < columnLimit)
-    {
-        output.append(columnLimit - column, L' ');
-        column = columnLimit;
-    }

```

Comment:
Removing this stops us from inserting padding spaces manually (that's the job of the terminal). But with the `if` condition gone, we need to move the `column = columnLimit;` elsewhere of course, because otherwise the caller doesn't know that we line-wrapped.